### PR TITLE
Add book redirection button (Book details 6)

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the back button in the book details. With this, when you enter in a book detail, now you can go back to the menu again. 

**New features**

- Add image back-arrow.png
- Set the image on the left, next to the "Atras" button. 
- The arrow image and the back button are clickeable and both redirects 

## Screenshots 
Screenshot from 2020-03-27 17-35-35

## Cards

https://trello.com/c/dbwSRcRc/33-detalle-de-libro-vi-redirecci%C3%B3n

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -9,6 +9,20 @@
...
 }
 
+.back-button {
+  margin: 30px;
+
+  &::after {
+  background: url("images/back-arrow-20.png") no-repeat;
+  content: url("index.html");
+  height: 25px;
+  position: absolute;
```
Comment (CrisVega08) [inline scss/detail.scss]: you should position arrow with back button now is with the viewport.

File: scss/detail.scss [hunk 2]
```diff
@@ -9,6 +9,20 @@
   display: flex;
 }
 
+.back-button {
+  margin: 30px;
+
+  &::after {
+  background: url("images/back-arrow-20.png") no-repeat;
```
Comment (CrisVega08) [inline scss/detail.scss]: check indentation

File: scss/detail.scss [hunk 3]
```diff
@@ -25,7 +39,7 @@
   background: url('images/badge.png') no-repeat;
   content: '';
   height: 120px;
-  left: 250px;
+  left: 500px;
```
Comment (CrisVega08) [inline scss/detail.scss]: Same here use position relative on .book-cover-row and take it like reference for absolute position.

File: scss/detail.scss [hunk 4]
```diff
@@ -44,6 +58,12 @@
   width: 200px;
 }
 
+.book-detail-container {
+  display: flex;
+  justify-content: center;
+  height: 768px;
```
Comment (CrisVega08) [inline scss/detail.scss]: I am not sure about .book-detail-container has a fixed height, why did you do it?

File: book-details.html [hunk 5]
```diff
@@ -11,24 +11,29 @@
...
-      </div>
-      <div class="book-content-row">
-        <h3 class="default-text category">Año de publicación:</h3>
-        <p class="default-text">Año de publicación</p>
+  <div class="book-detail-container">
+    <a href="./index.html">
+      <p class="back-button">Atrás</p>
+    </a>
+    <div class="book-cover-page">
+      <img class="book-img" src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline book-details.html]:
```suggestion
      <img class="book-img" src="images/book-cover.png" alt="Book Cover" />
```

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: book-details.html (hunk 5 above)
Comment (CrisVega08) [reply book-details.html]: Check scroll on details page, you should not have it

File: scss/detail.scss (hunk 2 above)
Comment (eledwinn) [reply scss/detail.scss]: Use single quotes in css please!

File: scss/detail.scss [hunk 6]
```diff
@@ -9,6 +9,20 @@
   display: flex;
 }
 
+.back-button {
+  margin: 30px;
+
+  &::after {
+  background: url("images/back-arrow-20.png") no-repeat;
+  content: url("index.html");
```
Comment (eledwinn) [inline scss/detail.scss]: Is this necessary?

**Review by eledwinn — CHANGES_REQUESTED** [review]
Please, check back button image, is not horizontally aligned.

Your development
image
 Invision
image

File: scss/detail.scss (hunk 4 above)
Comment (JuanGutierrez43) [reply scss/detail.scss]: That was unnecesary. I'll remove it :+1:

File: book-details.html (hunk 5 above)
Comment (JuanGutierrez43) [reply book-details.html]: The scroll was made by the fixed height that I used. When I remove that, the scroll just go away. Thank you!

File: scss/detail.scss [hunk 7]
```diff
@@ -9,6 +9,23 @@
...
 }
 
+.back-button {
+  display: flex;
+  justify-content: center;
+  margin: 30px;
+  font-size: 18px;
+  
+  &::before {
+  display: flex;
```
Comment (CrisVega08) [inline scss/detail.scss]: Remember indentation

File: scss/detail.scss [hunk 8]
```diff
@@ -9,6 +9,23 @@
   display: flex;
 }
 
+.back-button {
+  display: flex;
+  justify-content: center;
+  margin: 30px;
```
Comment (CrisVega08) [inline scss/detail.scss]: sort alphabetic

File: scss/detail.scss [hunk 9]
```diff
@@ -17,6 +34,7 @@
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-shadow;
   display: flex;
+  position: relative;
```
Comment (CrisVega08) [inline scss/detail.scss]: sort alphabetic

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 10]
```diff
@@ -9,6 +9,23 @@
   display: flex;
 }
 
+.back-button {
+  display: flex;
+  justify-content: center;
+  margin: 30px;
+  font-size: 18px;
```
Comment (CrisVega08) [inline scss/detail.scss]: sort alphabetic

File: scss/detail.scss [hunk 11]
```diff
@@ -25,9 +43,8 @@
   background: url('images/badge.png') no-repeat;
   content: '';
   height: 120px;
-  left: 250px;
+  left: 210px;
```
Comment (CrisVega08) [inline scss/detail.scss]: I think if you use right property the value decrease.

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: scss/detail.scss [hunk 12]
```diff
@@ -10,16 +10,19 @@
...
+  display: flex;
+  font-size: 18px;
+  justify-content: center;
   margin: 30px;
 
-  &::after {
-  background: url("images/back-arrow-20.png") no-repeat;
-  content: url("index.html");
+  &::before {
+  background: url('images/back-arrow-20.png') no-repeat;
```
Comment (eledwinn) [inline scss/detail.scss]: Fix indentation please

File: scss/detail.scss [hunk 13]
```diff
@@ -10,16 +10,19 @@
...
+  content: '';
+  display: flex;
   height: 25px;
-  position: absolute;
-  top: 25px;
-  left: 120px;
-  width: 25px;
+  margin-right: 5px;
+  position: relative;
+  width: 25px;;
```
Comment (eledwinn) [inline scss/detail.scss]:
```suggestion
  width: 25px;
```

File: scss/detail.scss [hunk 14]
```diff
@@ -32,16 +35,16 @@
   box-shadow: 4px 4px 10px 0 $black-shadow;
   display: flex;
   height: 416px;
+  position: relative;
   margin: 50px 0 0 50px;
   width: 900px;
   
   &::after {
   background: url('images/badge.png') no-repeat;
```
Comment (eledwinn) [inline scss/detail.scss]: Indent 😅

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: scss/detail.scss (hunk 14 above)
Comment (JuanGutierrez43) [reply scss/detail.scss]: Sorry :smile:

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**